core/src/backend/navigator.rs

```diff
@@ -20,6 +21,23 @@ pub enum NavigationMethod {
     Post,
 }
 
+/// The handling mode of navigate_to_url website calls.
+#[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
+#[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
+pub enum OpenURLMode {
+    /// Allow all navigate_to_url website calls.
+    #[serde(rename = "allow")]
+    Allow,
+
+    /// A confirmation dialogue opens with every navigate_to_url website call.
```

Comment:
I can change it, but just curious: why? Generally, the spelling dialogue is rather more common and not wrong.

Reply:
Ah, it's an American vs. British English thing. Well, up to you I guess, but I think we're using "dialog" throughout the rest of the project. Note the spelling on https://ruffle.rs/#usage:
![image](https://user-images.githubusercontent.com/71368227/231639857-cfc35894-1bc9-4f45-ad1c-07429adca5e1.png)

Reply:
Yeah, that's a point. I guess I'll adapt it to it then.
(Since it's already 5:30am for me, I think I'll add these changes and do the rebase tomorrow though.)